### PR TITLE
Fixed android save image issue

### PR DESCRIPTION
**Pull request by mistikk**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,10 +24,10 @@
   "dependencies": {
     "@babel/runtime": "^7.5.5",
     "@esteemapp/esteem-render-helpers": "^1.2.2",
-    "@ptomasroos/react-native-multi-slider": "^1.0.0",
-    "@react-native-community/cameraroll": "^1.3.0",
     "@esteemapp/react-native-autocomplete-input": "^4.2.1",
     "@esteemapp/react-native-multi-slider": "^1.1.0",
+    "@ptomasroos/react-native-multi-slider": "^1.0.0",
```
Comment (feruzm) [inline package.json]: This is not necessary, please remove this package, we have `@esteemapp/react-native-multi-slider` that replaces this.

File: src/components/postElements/body/view/postBodyView.js [hunk 2]
```diff
@@ -127,23 +128,73 @@ const PostBody = ({
...
+      Promise.resolve();
+    } catch (error) {
+      Promise.reject(error);
+    }
+  };
+
+  const _downloadImage = async uri => {
+    return RNFetchBlob.config({
+      fileCache: true,
+      appendExt: 'jpg',
```
Comment (feruzm) [inline src/components/postElements/body/view/postBodyView.js]: Do we have proper error handling if image meta type is different, like .gif or .png or .svg, etc.?

File: package.json (hunk 1 above)
Comment (feruzm) [reply package.json]: on recent PR merge, I have removed it, so if you remove on this branch, it won't conflict

Comment (mistikk) [reply package.json]: okay i will remove it

File: src/components/postElements/body/view/postBodyView.js (hunk 2 above)
Comment (mistikk) [reply src/components/postElements/body/view/postBodyView.js]: i tried with gif image it is working. Actually i added this config for an android bug

**Pull request merged**